test/parallel/test-assert.js

```diff
@@ -423,8 +423,7 @@ assert.throws(makeBlock(thrower, TypeError));
     assert.ok(e instanceof TypeError, 'type');
   }
   assert.strictEqual(true, threw,
-                     'a.throws with an explicit error is eating extra errors',
-                     a.AssertionError);
+                     'a.throws with an explicit error is eating extra errors');
```

Comment:
Not much of a story, it has always been there, nobody notices for 8 years
https://github.com/nodejs/node/commit/4f679fd8d01c11c908bd5ca439a7aff956cda4ad#diff-02063f30815b78f7986ee0c212140b8eR110